ci-operator/config/quarkus-qe/quarkus-test-suite/quarkus-qe-quarkus-test-suite-main__quarkus-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,49 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .openshift-interop/
+  to: quarkus-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tests:
+- as: quarkus-interop-aws
+  cron: 0 6 * * 1
+  remote_api: true
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "jira_epic":"LPTOCPCI-290"},
+          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic":"LPTOCPCI-290"},
+          {"step": "quarkus-execute-tests", "failure_type": "pod_failure", "classification": "Test Execution", "jira_project": "LPTOCPCI", "jira_epic":"LPTOCPCI-290"},
+          {"step": "quarkus-execute-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "LPTOCPCI", "jira_epic":"LPTOCPCI-290"}
```

Comment:
Shouldn’t this be going to PQE’s project?

Reply:
yes, I am checking with PQE on their Jira project and will update it

Reply:
Updated with PQE jira